utils/http.py

```diff
@@ -12,3 +12,15 @@ def retrieve_json(url, timeout=DEFAULT_TIMEOUT, verify=True):
     r = requests.get(url, timeout=timeout, verify=verify)
     r.raise_for_status()
     return r.json()
+
+# Get expvar stats
+def get_expvar_stats(key, port):
+    try:
+        json = retrieve_json("http://127.0.0.1:{port}/debug/vars".format(port=port))
```

Comment:
I think we should use `config['bind_host']` instead of `127.0.0.1` here. Could be passed as a parameter of this function